dev-libs/tlsh/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>t@laumann.xyz</email>
+		<name>Thomas Bracht Laumann Jespersen</name>
+	</maintainer>
+	<maintainer type="person" proxied="proxy">
+		<email>sam@gentoo.org</email>
+		<name>Sam James</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
@thesamesam I figured this was the right way to add metadata

Reply:
Yep!

Reply:
Nice!